? artprovider.patch
? build/.deps
? build/Makefile
? build/bk-deps
? build/config.log
? build/config.status
? build/configarg.cache
? build/contrib
? build/demos
? build/lib
? build/samples
? build/utils
? build/version-script
Index: include/wx/artprov.h
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/include/wx/artprov.h,v
retrieving revision 1.19
diff -b -u -2 -r1.19 artprov.h
--- include/wx/artprov.h	2004/11/27 23:23:44	1.19
+++ include/wx/artprov.h	2005/01/07 07:25:06
@@ -70,4 +70,6 @@
 #define wxART_GO_HOME              wxART_MAKE_ART_ID(wxART_GO_HOME)
 #define wxART_FILE_OPEN            wxART_MAKE_ART_ID(wxART_FILE_OPEN)
+#define wxART_FILE_SAVE            wxART_MAKE_ART_ID(wxART_FILE_SAVE)
+#define wxART_FILE_SAVE_AS         wxART_MAKE_ART_ID(wxART_FILE_SAVE_AS)
 #define wxART_PRINT                wxART_MAKE_ART_ID(wxART_PRINT)
 #define wxART_HELP                 wxART_MAKE_ART_ID(wxART_HELP)
@@ -92,4 +94,18 @@
 #define wxART_INFORMATION          wxART_MAKE_ART_ID(wxART_INFORMATION)
 #define wxART_MISSING_IMAGE        wxART_MAKE_ART_ID(wxART_MISSING_IMAGE)
+
+#define wxART_COPY                 wxART_MAKE_ART_ID(wxART_COPY)
+#define wxART_CUT                  wxART_MAKE_ART_ID(wxART_CUT)
+#define wxART_PASTE                wxART_MAKE_ART_ID(wxART_PASTE)
+#define wxART_DELETE               wxART_MAKE_ART_ID(wxART_DELETE)
+
+#define wxART_UNDO                 wxART_MAKE_ART_ID(wxART_UNDO)
+#define wxART_REDO                 wxART_MAKE_ART_ID(wxART_REDO)
+
+#define wxART_QUIT                 wxART_MAKE_ART_ID(wxART_QUIT)
+
+#define wxART_FIND                 wxART_MAKE_ART_ID(wxART_FIND)
+#define wxART_FIND_AND_REPLACE     wxART_MAKE_ART_ID(wxART_FIND_AND_REPLACE)
+
 
 // ----------------------------------------------------------------------------
Index: samples/artprov/artbrows.cpp
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/samples/artprov/artbrows.cpp,v
retrieving revision 1.7
diff -b -u -2 -r1.7 artbrows.cpp
--- samples/artprov/artbrows.cpp	2004/06/12 23:43:51	1.7
+++ samples/artprov/artbrows.cpp	2005/01/07 07:25:07
@@ -98,4 +98,15 @@
     ART_ICON(wxART_CROSS_MARK)
     ART_ICON(wxART_MISSING_IMAGE)
+	ART_ICON(wxART_FILE_SAVE)
+	ART_ICON(wxART_FILE_SAVE_AS)	
+	ART_ICON(wxART_COPY)
+	ART_ICON(wxART_CUT)
+	ART_ICON(wxART_PASTE)
+	ART_ICON(wxART_UNDO)
+	ART_ICON(wxART_REDO)
+	ART_ICON(wxART_QUIT)
+	ART_ICON(wxART_FIND)
+	ART_ICON(wxART_FIND_AND_REPLACE)
+
 }
 
Index: src/gtk/artgtk.cpp
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/src/gtk/artgtk.cpp,v
retrieving revision 1.5
diff -b -u -2 -r1.5 artgtk.cpp
--- src/gtk/artgtk.cpp	2004/12/05 23:03:09	1.5
+++ src/gtk/artgtk.cpp	2005/01/07 07:25:10
@@ -103,4 +103,20 @@
     ART(wxART_HARDDISK,                            GTK_STOCK_HARDDISK)
     ART(wxART_REMOVABLE,                           GTK_STOCK_HARDDISK)
+
+	ART(wxART_FILE_SAVE,                           GTK_STOCK_SAVE)
+	ART(wxART_FILE_SAVE_AS,                        GTK_STOCK_SAVE_AS)
+
+	ART(wxART_COPY,                                GTK_STOCK_COPY)
+	ART(wxART_CUT,                                 GTK_STOCK_CUT)
+	ART(wxART_PASTE,                               GTK_STOCK_PASTE)
+	ART(wxART_DELETE,                              GTK_STOCK_DELETE)
+
+	ART(wxART_UNDO,                                GTK_STOCK_UNDO)
+	ART(wxART_REDO,                                GTK_STOCK_REDO)
+
+	ART(wxART_QUIT,                                GTK_STOCK_QUIT)
+
+	ART(wxART_FIND,                                GTK_STOCK_FIND)
+	ART(wxART_FIND_AND_REPLACE,                    GTK_STOCK_FIND_AND_REPLACE)
 #endif
     

 	  	 
